lms/djangoapps/grades/tests/data/capa.xml

```diff
@@ -0,0 +1,6 @@
+<problem url_name="capa-optionresponse">
```

Comment:
How about a new `openedx.core.lib.xblock_utils.test_utils` file?  Seems like the closest match of what I can find right now.